jupyter-js-widgets/src/widget_int.js

```diff
@@ -200,23 +203,43 @@ var IntSliderView = widget.DOMWidgetView.extend({
             var readout = this.model.get('readout');
             if (readout) {
                 this.readout.style.display = '';
+                window.setTimeout(function() {
```

Comment:
Hum, wondering if I need to delay this anymore. I was computing overflow differently earlier.